ES: I need a dumb topic name

maxii1996 opened the pull request.

maxii1996 committed:
Merge pull request #7 from Arks-Layer/ES

Merge pull request #892 from maxii1996/ES

maxii1996 committed:
Super Update

:D

maxii1996 committed:
Update co_058010.csv

maxii1996 committed:
Actualizar

maxii1996 commented:
IDK why gives me error D:

maxii1996 committed:
Revert "Super Update"

This reverts commit edb984093adf09025f53b780fe72a5eb799c0b75.

maxii1996 committed:
Update co_058010.csv

maxii1996 committed:
ES probemos ahi

alama commented:
Because you changed some of the CSVs in the Files folder? you need to move them out

maxii1996 commented:
> Because you changed some of the CSVs in the Files folder? you need to move them out

I dont know what i've done D:

maxii1996 committed:
Delete ac06040411.csv

maxii1996 committed:
Delete quest_00020180.csv

maxii1996 committed:
Delete quest_00130040.csv

alama commented:
No, you do not need to delete them, just rename/move them

maxii1996 commented:
but i dont know what the real name is it. And why it change it?
So, I'll start again

maxii1996 committed:
Delete co_058010.csv

maxii1996 committed:
Delete quest_00020240.csv

maxii1996 committed:
Delete quest_00020260.csv

alama committed:
Revert "Delete quest_00020260.csv"

This reverts commit 740823d4acb057a782975609664e1244ad742af9.

alama committed:
Revert "Delete quest_00020240.csv"

This reverts commit bf3fcdf60db5bd348e36d50676f7a27510ec7e4b.

alama committed:
Revert "Delete co_058010.csv"

This reverts commit f5a5636db9bf6de124d455ecf984a05d1fe67e18.

alama committed:
Revert "Delete quest_00130040.csv"

This reverts commit 4be40451493125f968217d0615fc99af7c6d6039.

alama committed:
Revert "Delete quest_00020180.csv"

This reverts commit 2552f874a717b1c0ec957f8e22c00d7f51c6f88a.

alama committed:
Revert "Delete ac06040411.csv"

This reverts commit 56b7dcf6f5b77a6fc8cc71acde9d1141ee7dca1e.

alama committed:
Moved that changed files to Files.TODO

alama commented on Files.TODO/co_058010.csv:
```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
 Title#0,"""『領域調査：異世界の残滓の解放』について"""
+=======
+Title#0,"""Inspec. Restos del Otro mundo"""
```
Why are there 2 `Title#0`?

alama committed:
Undo mismerge from maxii1996's mess

maxii1996 commented:
Thanks you @alama :D

I'm really noob using GitHub i guess xD

alama reviewed (changes requested):
Can not be merged until the CSV files in Files.TODO folder had been moved into suitable folders

maxii1996 commented:
@TheBlazter Por favor, dame una mano. Tengo miedo de hacer mas lio D:

maxii1996 committed:
ES: Sincerente no se lo que hago

Sinceramente no se bien lo que tenia que hacer... Movi los archivos de la Carpeta Files.TODO a las carpetas que me dijiste @TheBlazter pero no estoy seguro que haya sido lo que me pedian D:

maxii1996 commented:
ES: Creo que ya está? 
EN: I think is done?

alama committed:
Merge tag 'ES-v60501_rc_19' into maxii1996

TheBlazter commented:
Thanks @alama for the merge
